docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -0,0 +1,33 @@
+| Console Command | The `CONSOLE` and `CONSPROF` commands must exist in the authorized command table. |
+| Java version | IBM 64-bit SDK for z/OS, Java Technology Edition V7.1 or higher is required. However, we experienced problems accessing z/OSMF 2.2 using Java version 8. If you use z/OSMF 2.3, Java version 8.0_64 is required. <br/>```SYS1.PARMLIB member  IZUPRMxx.... JAVA_HOME('/sys/java64bt/v7r1m0/usr/lpp/java/J7.1_64'/etc/profile ............................................... export JAVA_HOME=/sys/java31bt/v7r1m0/usr/lpp/java/J7.1 ```<br/>For more information, see [Software prerequisites for z/OSMF](https://www.ibm.com/docs/en/zos/2.5.0?topic=zosmf-software-prerequisites) on the IBM Documentation. |
+| Maximum region size | To prevent exceeds maximum region size errors, ensure that you have a TSO maximum region size of at least 65536 KB for the z/OS system. |
+| User IDs | User IDs require a TSO segment (access) and an OMVS segment. During workflow processing and REST API requests, z/OSMF may start one or more TSO address spaces under the following job names: <ul><li>*userid*</li><li>*substr(userid, 1, 6)//CN* (Console)</li></ul> Example: ```(userid = USRMY01, USRMY0CN )```
+
+## Selecting and configuring your z/OSMF plug-ins
+
+| Plug-in        | Functionality | Task |
+| ----------- | ----------- | ----------- |
+| (Optional) Cloud Portal | The Cloud Portal plug-in lets you make software services available to marketplace consumers and it adds the Marketplace and Marketplace Administration tasks to the z/OSMF navigation tree. | For information about how to enable the plug-in, see [Cloud provisioning marketplace](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-cloud-provisioning-marketplace) on the IBM Documentation. |
+| Configuration Assistant | The Configuration Assistant plug-in lets z/OSMF configure TCP/IP policy-based networking functions.<br/>For more information about the functionality that the plug-in provides, see [Configuration Assistant task overview](https://www.ibm.com/docs/en/zos/2.2.0?topic=tasks-configuration-assistant-task-overview) on the IBM Documentation. | For information about how to enable the plug-in, see Updating z/OS for the [Configuration Assistant plug-in](https://www.ibm.com/docs/en/zos/2.2.0?topic=ins-updating-zos-configuration-assistant-plug-in) on the IBM Documentation. |
+| ISPF | The ISPF plug-in lets z/OSMF access traditional ISPF applications.<br/>For information about the functionality that the plug-in provides, see [ISPF task overview](https://www.ibm.com/docs/en/zos/2.2.0?topic=tasks-ispf-task-overview) on the IBM Documentation. | For information about how to enable the plug-in, see [Updating z/OS for the ISPF service](https://www.ibm.com/docs/en/zos/2.5.0?topic=service-updating-zos-ispf) on the IBM Documentation. |
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for the v2.2 article ("Updating z/OS for the ISPF service") in the IBM Docs. Could one of you help me find a replacement?

Reply:
Please verify:
https://www.ibm.com/docs/en/zos/2.5.0?topic=interfaces-ispf

Reply:
verifying... 😋 

Yeah this is probably a better overview than what they had in the 2.2.0 version. Lots of links to other doc pages, but does the job 😋